content/_code-samples/escrow/py/finish_escrow.py

```diff
@@ -0,0 +1,42 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import EscrowFinish
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to the testnetwork
+
+# Complete an escrow
+# Cannot be called until the finish time is reached
+
+
+escrow_creator = generate_faucet_wallet(client=client).classic_address
+
+escrow_sequence = 27641268
+
+# Optional fields
+
+# Cryptio condition that must be met before escrow can be completed, passed on escrow creation
+condition = "A02580203882E2EB9B44130530541C4CC360D079F265792C4A7ED3840968897CB7DF2DA1810120"
+
+# Cryptio fulfillment of the condtion 
```

Comment:
Can you add a link explaining how to generate this? (Or to the spec which explains it?)

Also small typo 
```suggestion
# Crypto fulfillment of the condtion 
```

Reply:
https://github.com/ObiajuluM/xrpl-dev-portal/blob/master/content/_code-samples/escrow/py/generate_condition.py

I created a special file to show how to generate the condition

Reply:
https://github.com/ObiajuluM/xrpl-dev-portal/blob/master/content/_code-samples/escrow/py/generate_condition.py

There is a separate file that shows how to generate this